packages/api/test/addbuild-getbuild.js

```diff
@@ -136,6 +136,7 @@ describe('Add-Build', () => {
 
       // ensure builds were uploaded correctly
       const builds = await client.collection(global.headCollection).doc(buildInfo[1].repoId).collection('builds').doc(buildInfo[1].buildId).get();
+      console.log('BUIDLS>DATA: ' + builds.data().percentpassing);
```

Comment:
is this a useful log message to have in for now? or  accidentally left. Happy to leave the log line if it's helping during debugging.